app/packages/files/file.ts

```diff
@@ -46,8 +52,12 @@ export default abstract class File {
     providerHandlesVersioning = true;
 
     currentUser: CurrentUserService;
+    @service intl!: Intl;
+    @service toast!: Toast;
+
 
     constructor(currentUser: CurrentUserService, fileModel: FileModel) {
+        setOwner(this, getOwner(fileModel));
```

Comment:
@futa-ikeda This is the main difference between the previous version of this PR and the new one. POJOs need to have their owner set in order to use services, so I set the owner here and in provider-file.